src/index.ts

```diff
@@ -42,12 +41,14 @@ export async function calculateRateFromPath(paths: ConversionPaths, amount) {
     return amount;
 }
 
-async function getConverterPairs(path: string[], blockchainType: BlockchainType) {
+async function getConverterPairs(path: string[] | object[], blockchainType: BlockchainType) {
```

Comment:
generic code, shouldn't have string or object here - I thought we're going to define Converter objects?